<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Loyolas-MBP" failures="1" tests="5" name="Surefire test" time="106.693" errors="0" timestamp="2023-05-02T09:40:07 GMT-04:00">
  <testcase classname="rahulshettyacademy.tests.ErrorValidationTest" name="loginErrorValidationTest" time="0.479"/>
  <testcase classname="rahulshettyacademy.tests.ErrorValidationTest" name="productErrorValidationTest" time="19.821"/>
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="submitOrder" time="21.767"/>
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="submitOrder" time="22.053"/>
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="OrderHistoryTest" time="10.625">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.tests.SubmitOrderTest.OrderHistoryTest(SubmitOrderTest.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Surefire test -->
